### DISCUSSION TITLE
zfs create

### DISCUSSION THREAD

**Discussion by bananabrick** [openzfs/zfs#12396] [General]
If I run something like `zpool create -f <name> <dir> <drive>`, then it seems as if the `<dir>` folder allows me to access the `<drive>`, and the underlying file system is `zfs`.

Is there any reason to `zfs create`, to create subdirectories/subfile systems, inside `<dir>`, afterwards?

**Comment by allanjude** [comment]
I am not sure what you are asking.

That is not a valid syntax for `zpool create`, the `-f` flag just overrides the safety warnings about your drive selection

so the syntax should be `zpool create <poolname> <optional-vdev-type> <drive1> <drive2> <optional-vdev-type> <drive3> <drive4>`

There is no 'dir', If you wish to set the default mountpoint for the pool, that is the `-m <dir>` flag.

**Reply by bananabrick** [reply]
My bad. What I meant was, if I run zpool create -f <name> -m <mountpoints> <drive_name>, is there any reason to run `zfs create`, afterwards?
